src/viking.js

```diff
@@ -1,11 +1,71 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength) {
+        this.health= health;
+        this.strength =strength;
+    }
+    attack() {
+    return  this.strength;
+    }
+    receiveDamage(theDamage){
+        this.health -= theDamage;
+    }
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name, health, strength) {
+        super(health, strength)
+        this.name = name
+    }
+    attack() {
+    return  this.strength;
+    }
+    receiveDamage(theDamage){
+        this.health -= theDamage;
+        if (this.health> 0){
+            return `${this.name} has received ${theDamage} points of damage`
+        }
+        else {
+            return `${this.name} has died in act of combat`
+        }
+    } battleCry(){
+        return "Odin Owns You All!"
+    }
+}
+
+
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+    constructor(health, strength) {
+        super(health, strength)
```

Comment:
Since `Saxon` is extending the class `Soldier` and the only thing that is different in the `Saxon` class is the `receiveDamage()` method you don't need the `constructor` neither the `super` here:

```
class Saxon extends Soldier {
  receiveDamage(theDamage) {
    this.health -= theDamage;
    if (this.health > 0)
      return `A Saxon has received ${theDamage} points of damage`;
    else return `A Saxon has died in combat`;
  }
}
```